packages/bitcoin/redux/reducers.js

```diff
@@ -3,12 +3,169 @@
  * Licensed under the AGPL Version 3 license.
  */
 
+import * as ActionsTypes from './constants'
 import initialState from './initialState'
 
-const mutations = {}
+const mutations = {
 
-export default (state = initialState, { type, ...payload }) => {
+  [ActionsTypes.BITCOIN_CREATE_WALLET]: (state, { address }) => ({
+    ...state,
+    list: {
+      address,
```

Comment:
This reducer will erase all other wallets each time. We need to append wallets into the list.
List of BitCoin wallets should has Ethereum address as a key:
```
    list: {
      "10005RegqPWqJSxmhEGoGGakFFGsiDZ63G": {   <<< This is Ethereum address
        address: "1005RegqPWqJSxmhEGoGGakFFGsiDZ63G", << Bitcoin address
        parentWalletAddress: "10005RegqPWqJSxmhEGoGGakFFGsiDZ63G", << Duplication required
        transactions: {},
        pending: null,
        tokens: {
          'BTC': { // Constant
            symbol: 'BTC',
            decimals: number, // Here we need to use `const DECIMALS = 8` constant from package/bitcoin/constants
          }
        }
      },
      "11005RegqPWqJSxmhEGoGGakFFGsiDZ63G": { << Another Ethereum address, like another main wallet
        address: "11005RegqPWqJSxmhEGoGGakFFGsiDZ63G",
        parentWalletAddress: "11005RegqPWqJSxmhEGoGGakFFGsiDZ63G",
        transactions: {},
        pending: null,
      },
    },
``` 

Also I've added all preliminary store structure into description of CM-246 user story (in active sprint)